packages/core/src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -212,6 +212,7 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
             const state = await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(options.methodId));
             const billing = state.billingAddress.getBillingAddressOrThrow();
             const customer = state.customer.getCustomerOrThrow();
+            const {features} = state.config.getStoreConfigOrThrow().checkoutSettings;
```

Comment:
```suggestion
            const { features } = state.config.getStoreConfigOrThrow().checkoutSettings;
```

Reply:
Hi @animesh1987, I do the change, could you please take another look?